docs/FeatureManagement/Reference.md

```diff
@@ -0,0 +1,13 @@
+# Azure App Configuration Feature Management
```

Comment:
Should we name this readme.md so people will see this doc automatically when navigate to the FeatureManagement folder?

Reply:
Good idea. Done.